site/en/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'User-Agent Snippets'
```

Comment:
```suggestion
title: 'User-Agent reduction snippets'
```

Reply:
I like the snappy tittle, it makes sense when you navigate to it via the landing page but this might be more SEO friendly for search and repeat visitor. At your discretion.

Reply:
Updating.